l3build.dtx

```diff
@@ -1887,32 +1893,32 @@
 %
 % \subsection{Components of \texttt{l3build}}
 %
+% Next functions return $0$ on a successful completion, a non $0$ error code otherwise.
```

Comment:
I'm not sure what this is referring to here in 'next'

Reply:
The functions documented in the section?

I do not really understand the title of the section, as compared to the following section.